data_steward/cdr_cleaner/cleaning_rules/remove_operational_pii_fields.py

```diff
@@ -0,0 +1,239 @@
+        into project_id.sandbox_dataset_id.operational_pii_fields in BQ
+        """
+        table_path = os.path.join(resources.resource_files_path,
+                                  f"{OPERATIONAL_PII_FIELDS_TABLE}.csv")
+        with open(table_path, 'rb') as csv_file:
+            schema_list = client.get_table_schema(OPERATIONAL_PII_FIELDS_TABLE)
+            table_id = f'{self.project_id}.{self.sandbox_dataset_id}.{OPERATIONAL_PII_FIELDS_TABLE}'
+            job_config = bigquery.LoadJobConfig(
+                schema=schema_list,
+                skip_leading_rows=1,
+                source_format=bigquery.SourceFormat.CSV,
+                field_delimiter=',',
+                allow_quoted_newlines=True,
+                quote_character='"',
+                write_disposition=bigquery.job.WriteDisposition.WRITE_TRUNCATE)
+
+            # job_id defined to the second precision
+            job_id = f'{self.dataset_id}_{self.__class__.__name__}_{datetime.now().strftime("%Y%m%d_%H%M%S")}'
+            LOGGER.info(f'Loading `{table_id}`')
+            try:
+                load_job = client.load_table_from_file(
+                    csv_file, table_id, job_config=job_config,
+                    job_id=job_id)  # Make an API request.
+
+                load_job.result()  # Waits for the job to complete.
+            except (ValueError, TypeError) as exc:
+                LOGGER.info(
+                    'Something went wrong and the table did not load correctly')
+                raise exc
+            else:
+                LOGGER.info(f'Loading of `{table_id}` completed.')
+
+    def get_query_specs(self, *args, **keyword_args) -> query_spec_list:
+        """
+        Return a list of dictionary query specifications.
+
+        :return:  A list of dictionaries. Each dictionary contains a single query
+            and a specification for how to execute that query. The specifications
+            are optional but the query is required.
+        """
+        queries_list = []
+
+        sandbox_query = dict()
+        sandbox_query[cdr_consts.QUERY] = SANDBOX_OPERATIONAL_PII_FIELDS.render(
+            project_id=self.project_id,
+            dataset_id=self.dataset_id,
+            sandbox_dataset_id=self.sandbox_dataset_id,
+            operational_pii_fields_table=OPERATIONAL_PII_FIELDS_TABLE,
+            intermediary_table=INTERMEDIARY_TABLE)
+        queries_list.append(sandbox_query)
+
+        delete_query = dict()
+        delete_query[cdr_consts.QUERY] = DELETE_QUERY.render(
+            project_id=self.project_id,
+            dataset_id=self.dataset_id,
+            sandbox_dataset_id=self.sandbox_dataset_id,
+            intermediary_table=INTERMEDIARY_TABLE)
+        queries_list.append(delete_query)
+
+        return queries_list
+
+    def setup_validation(self, client: BigQueryClient) -> None:
+        """
+        Run required steps for validation setup
+        """
+        self.init_counts = self._get_counts(client)
```

Comment:
`self.init_counts` seems to be a local variable since I don't see it defined in the constructor. So, you should update this line to `init_counts = self._get_counts(client)`